src/pages/en/guides/server-side-rendering.md

```diff
@@ -132,3 +139,75 @@ export async function get({ params }) {
   });
 }
 ```
+
+While in this example, a valid HTML code can be returned to render the page or it's content, API Routes can be used as REST API Endpoints.
+
+### Serverless Functions
+
+API Routes can be used to run [serverless functions](https://en.wikipedia.org/wiki/Serverless_computing#Serverless_runtimes) without exposing sensitive data to the clients, such as authentications, database access, verifications, etc.
+
+In the example below, an API Route is used to verify a Google reCaptcha v3 without exposing the site-secret to the clients.
```

Comment:
```suggestion
In the example below, an API route is used to verify Google reCaptcha v3 without exposing the site-secret to the clients.
```
I'm not *exactly* sure what the correct phrasing is here, but from reading around, it doesn't seem common to say "a Google reCaptcha" like a singular noun. Is it a "Google reCaptcha *instance*?" a "Google reCaptcha *execution*?" Does it sound correct to just say it without the "a" entirely?

Or can the sentence be reworded to something like, "See below for an example of adding Google reCaptcha v3 to your Astro site and using an API route for verifcation without exposing the site-secret to the client." ?